hls4ml/writer/vivado_accelerator_writer.py

```diff
@@ -341,6 +347,19 @@ def write_driver(self, model):
     def write_new_tar(self, model):
         os.remove(model.config.get_output_dir() + '.tar.gz')
         super(VivadoAcceleratorWriter, self).write_tar(model)
+
+    def modify_project_tcl(self, model):
+        oldfile = open('{}/project.tcl'.format(model.config.get_output_dir()), 'r')
+        newfile = open('{}/project.temp.tcl'.format(model.config.get_output_dir()), 'w')
+        for l in oldfile.readlines():
+            if "backend \"vivado\"" in l:
+                l = l.replace("vivado", "vivadoaccelerator")
+            newfile.write(l)
+
+        oldfile.close()
+        newfile.close()
+        os.rename('{}/project.temp.tcl'.format(model.config.get_output_dir()), '{}/project.tcl'.format(model.config.get_output_dir()))
+
```

Comment:
This no longer works, I think because `write_board_script` creates a new `project.tcl`, and it's called later. I think the fix is to get rid of this function and just write the necessary lines in `write_board_script`.